core/src/main/java/com/datastax/oss/driver/api/core/metadata/Node.java

```diff
@@ -15,6 +15,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/*
+ * Copyright (C) 2023 ScyllaDB
```

Comment:
2023 here, 2024 elsewhere?

Reply:
Whenever I noticed it's missing i tried to apply the earliest year I could see in git blame